airsonic-main/src/test/java/org/airsonic/player/service/MediaScannerServiceTestCase.java

```diff
@@ -161,12 +148,11 @@ public void testScanLibrary() {
 
     @Test
     public void testSpecialCharactersInFilename() throws Exception {
-        Resource resource = resourceLoader.getResource("MEDIAS/piano.mp3");
         String directoryName = "Muff1nman\u2019s \uFF0FMusic";
         String fileName = "Muff1nman\u2019s\uFF0FPiano.mp3";
         File artistDir = temporaryFolder.newFolder(directoryName);
         File musicFile = artistDir.toPath().resolve(fileName).toFile();
-        IOUtils.copy(resource.getInputStream(), new FileOutputStream(musicFile));
+        Resources.copy(Resources.getResource("MEDIAS/piano.mp3"), new FileOutputStream(musicFile));
```

Comment:
This is irrelevant to the Spring bump, and also unstable.

Reply:
This is a Guava @Beta annotated class, not "unstable". Please read the javadocs for the annotation for when and when not to use this class.